skbio/stats/distance/_base.py

```diff
@@ -55,7 +55,9 @@ class DissimilarityMatrix(SkbioObject, PlottableMixin):
 
     Methods are provided to load and save dissimilarity matrices from/to disk,
     as well as perform common operations such as extracting dissimilarities
-    based on object ID.
+    based on object ID. Additionally, the
+    :func:`~skbio.stats.distance.DissimilarityMatrix.plot` method provides
```

Comment:
I think that ":func:`~skbio.stats.distance.DissimilarityMatrix.plot`" can simply be ":meth:`plot`"